sources/packages/backend/apps/api/src/route-controllers/designation-agreement/models/designation-agreement.dto.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  DesignationAgreementStatus,
+  NOTE_DESCRIPTION_MAX_LENGTH,
+} from "@sims/sims-db";
+import { Type } from "class-transformer";
+import {
+  IsArray,
+  IsDate,
+  IsEnum,
+  IsNotEmpty,
+  IsOptional,
+  MaxLength,
+  ValidateNested,
+} from "class-validator";
+import { UpdateDesignationLocation } from "./designation-agreement.model";
+
+/**
+ * Approve/Deny a designation agreement.
+ * startDate, endDate and locationsDesignations used only for approval.
+ */
+export class UpdateDesignationAPIInDto {
+  @IsEnum(DesignationAgreementStatus)
+  designationStatus: DesignationAgreementStatus;
+  @IsOptional()
+  @IsDate()
+  startDate?: Date;
```

Comment:
These dates are probably date-only, if I am not wrong and they should be mapped as strings with `@IsDateString`.